### PR TITLE
Update sequence diagrams: add encryption for refresh address to the flow

### PR DESCRIPTION
**Pull request by mpanne**
and move generation of QR code from backend to widget.

### CODE REVIEW

File: doc/eID/eid-flow-paula.md [hunk 1]
```diff
@@ -21,38 +21,44 @@ sequenceDiagram
...
     user ->> app: approve data request
     app ->> smartphone: access NFC reader
     smartphone ->> id: scan NFC chip
-    id -->> smartphone: return [encrypted identity data]
-    app ->> server: send [encrypted identity data]
-    server ->> server: decrypt [encrypted identity data] 
+    id -->> smartphone: return <br>[encrypted identity data]
+    smartphone -->> app: return <br>[encrypted identity data]
+    app ->> server: send <br>[encrypted identity data]
+    server ->> server: decrypt <br>[encrypted identity data]
```
Comment (code28) [inline doc/eID/eid-flow-paula.md:50]: Since we decrypt here, shouldn't it be `decrypted identity data`?

File: doc/eID/eid-flow-paula.md [hunk 2]
```diff
@@ -21,38 +21,44 @@ sequenceDiagram
...
+    widget ->> widget: decrypt refreshURL with widgetSessionSecret <br>[refreshURL]
+    widget ->> browser: redirect to <br>[refreshAddress]
     browser ->> eService: access success page
-    eService ->> backend: get identity data [sessionId]
-    backend ->> server: get identity data [sessionId]
-    server -->> backend: return [decrypted identity data]
-    backend -->> eService: return [decrypted identity data]
+    eService ->> backend: get identity data <br>[sessionId]
+    backend ->> server: get identity data <br>[sessionId]
+    server -->> backend: return <br>[decrypted identity data]
```
Comment (code28) [inline doc/eID/eid-flow-paula.md:60]: Actually, we don't need the keyword `decrypted` here. It's more descriptive, but then we could add it at `refresh address` as well? For me both is fine.

File: doc/eID/eid-flow-magnus.md [hunk 3]
```diff
@@ -20,35 +20,41 @@ sequenceDiagram
...
     user ->> app: approve data request
     app ->> smartphone: access NFC reader
     smartphone ->> id: scan NFC chip
-    id -->> smartphone: return [encrypted identity data]
-    app ->> server: send [encrypted identity data]
-    server ->> server: decrypt [encrypted identity data]
+    id -->> smartphone: return <br>[encrypted identity data]
+    smartphone -->> app: return <br>[encrypted identity data]
+    app ->> server: send <br>[encrypted identity data]
+    server ->> server: decrypt <br>[encrypted identity data]
```
Comment (code28) [inline doc/eID/eid-flow-magnus.md:48]: Same as Paula: `decrypted identity data`

**Review by code28 — CHANGES_REQUESTED** [review]

File: doc/eID/eid-flow-paula.md (hunk 2 above)
Comment (mpanne) [reply doc/eID/eid-flow-paula.md:60]: I agree. I will remove the "decrypted" to keep it lean.

**Review by code28 — APPROVED** [review]

**Pull request merged**